src/Esfa.Vacancy.Register.Api/Mappings/RecruitApprenticeshipMapper.cs

```diff
@@ -70,9 +70,9 @@ public async Task<ApiTypes.ApprenticeshipVacancy> MapFromRecruitVacancy(recruitE
                 EmployerName = liveVacancy.EmployerName,
                 EmployerDescription = liveVacancy.EmployerDescription,
                 EmployerWebsite = liveVacancy.EmployerWebsiteUrl,
-                ContactName = liveVacancy.EmployerContactName,
-                ContactEmail = liveVacancy.EmployerContactEmail,
-                ContactNumber = liveVacancy.EmployerContactPhone,
+                ContactName = liveVacancy.EmployerContactName ?? liveVacancy.ProviderContactName,
+                ContactEmail = liveVacancy.EmployerContactEmail ?? liveVacancy.ProviderContactEmail,
+                ContactNumber = liveVacancy.EmployerContactPhone ?? liveVacancy.ProviderContactPhone,
```

Comment:
If EmployerContactName etc can be entered in a web page then they could be string.Empty. If that is the case you should check for string.IsNullOrWhiteSpace instead.

Reply:
Good point but from the source the object we are mapping from, we do not store empty strings therefore it will either be a null or non empty string but not whitespaced as we do white space trimming before we validate/save. Also the employer/provider contact details would not be set therefore null representation in the C# object.

Reply:
Should we create Provider contact properties rather than reuse the existing ones? FAA has these split so it could handle both employer and provider contact info if needed.

Reply:
Would think that would be more appropriate if we had a v2(not v2 system) version of this endpoint. Currently all v1 system vacancies have provider contact details as they are created by provider, can't really map them into a ProviderContact* fields on this response even though it is additive. Also duplicates in the Contact* fields will cause confusion as V1 vacancies will have six fields and v2 provider vacancies will have ProviderContact* fields but really the Contact* fields will be obsolete and make responses look inconsistent without the user knowing whether the response is for a V1 or v2 vacancy which shouldn't really make a difference from this API (yet).

If we did a v2 endpoint then we could have EmployerContact* fields where applicable and ProviderContact* fields where applicable.

On FAA, I believe two different view models are being used (one for v1 vacancies and another for v2 recruit vacancies). I know the page has different headings but don't want to overcomplicate this API response.